user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -824,5 +834,230 @@ function testCasesForPolicyTable:Restore_preloaded_pt()
 		commonPreconditions:RestoreFile(pt_fileName)
 	end
 end	
-	
+
+-- The funcion will be used when PTU is triggered. 
+-- 1. It is assumed that notification is recevied: EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+-- 2. It is assumed that request/response is received: EXPECT_HMICALL("BasicCommunication.PolicyUpdate", 
+-- 3. Function will use default endpoints
+function testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY(app_id, device_id, hmi_app_id, ptu_file_path, ptu_file_name)
+	function Test:TestStep_Flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
+		if (app_id == nil) then app_id = config.application1.registerAppInterfaceParams.appID end
+		if (device_id == nil) then device_id = config.deviceMAC end
+		if (hmi_app_id == nil) then hmi_app_id = self.applications[config.application1.registerAppInterfaceParams.appName] end 
+		if (ptu_file_path == nil) then ptu_file_path = "files/" end
+		if (ptu_file_name == nil) then ptu_file_name = "PolicyTableUpdate" end
```

Comment:
@Itileda The idea of function is to be used as precondition for other steps. The result of this function should be UP_TO_DATE. The idea of the parameters is if tested knows that is using correct policy file, to be able to use. If tester wants to use default parameters and just to have result code UP_TO_DATE the default parameters will be used.